app/equip/potion.py

```diff
@@ -0,0 +1,12 @@
+class Potion:
+    def __init__(self, potion: dict):
+        self.potion = potion
+
+    def get_potion(self, hero) -> None:

```

Comment:
fix annotations everywhere

Reply:
How to use annotation to hero. If i import Hero class i will get an error about circular import

Reply:
yeah, sry, didn't notice